bin/crew

```diff
@@ -124,6 +124,25 @@ String.use_color = args['--color'] || !args['--no-color']
 @verbose           = @opt_verbose ? 'v' : ''
 @short_verbose     = @opt_verbose ? '-v' : ''
 
+class ExitMessage
+  attr_accessor :msg
+
+  @obj_all = []
+
+  def initialize(msg)
```

Comment:
This can be deleted as the `.new` function has been overwritten

Reply:
You're saying I don't need lines 129-132?

Reply:
The `def initialize` function can be deleted (from line 132 to 134)